test/ServerComparison.FunctionalTests/NtlmAuthenticationTest.cs

```diff
@@ -70,66 +71,45 @@ public NtlmAuthenticationTests(ITestOutputHelper output) : base(output)
-                        */
+
                         logger.LogInformation("Enabling Default Credentials");
 
                         // Change the http client to one that uses default credentials
                         httpClient = deploymentResult.CreateHttpClient(new HttpClientHandler() { UseDefaultCredentials = true });
 
-                        logger.LogInformation("Testing /AutoForbid");
-                        response = await httpClient.GetAsync("/AutoForbid");
+                        logger.LogInformation("Testing /Challenge");
+                        response = await httpClient.GetAsync("/Challenge");
                         responseText = await response.Content.ReadAsStringAsync();
                         Assert.Equal(HttpStatusCode.Forbidden, response.StatusCode);
 
                         logger.LogInformation("Testing /Restricted");
                         response = await httpClient.GetAsync("/Restricted");
                         responseText = await response.Content.ReadAsStringAsync();
                         Assert.Equal(HttpStatusCode.OK, response.StatusCode);
-                        Assert.Equal("Negotiate", responseText);
-
-                        logger.LogInformation("Testing /RestrictedNegotiate");
-                        response = await httpClient.GetAsync("/RestrictedNegotiate");
-                        responseText = await response.Content.ReadAsStringAsync();
-                        Assert.Equal(HttpStatusCode.OK, response.StatusCode);
-                        Assert.Equal("Negotiate", responseText);
 
-                        logger.LogInformation("Testing /RestrictedNTLM");
                         if (serverType == ServerType.WebListener)
                         {
-                            response = await httpClient.GetAsync("/RestrictedNTLM");
-                            responseText = await response.Content.ReadAsStringAsync();
-                            Assert.Equal(HttpStatusCode.OK, response.StatusCode);
-                            Assert.Equal("NTLM", responseText);
+                            Assert.Equal("Negotiate", responseText);
                         }
                         else if (serverType == ServerType.IISExpress)
                         {
-                            response = await httpClient.GetAsync("/RestrictedNTLM");
-                            responseText = await response.Content.ReadAsStringAsync();
-                            // This isn't a Forbidden because we authenticate with Negotiate and challenge for NTLM.
-                            Assert.Equal(HttpStatusCode.Unauthorized, response.StatusCode);
-                            // Note IIS can't restrict a challenge to a specific auth type, the native auth modules always add themselves,
-                            // so both Negotiate and NTLM get sent again.
+                            Assert.Equal("Kerberos", responseText);
```

Comment:
This auth type is unstable, just check !IsNullOrEmpty, then you can remove the server specific checks